lib/ansible/plugins/inventory/script.py

```diff
@@ -162,6 +162,7 @@ def parse(self, inventory, loader, path, cache=None):
 
     def _parse_group(self, group, data):
 
+        group = to_safe_group_names(group)
```

Comment:
unsure if this is correct fix or should add_group/add_host return 'actual name used' as a cleaner api?